pkg/k8s/apis/cilium.io/v2/validator/validator.go

```diff
@@ -105,10 +110,97 @@ func (n *NPValidator) ValidateCNP(cnp *unstructured.Unstructured) error {
 	return nil
 }
 
+var (
+	// We can remove the check for this warning once 1.9 is the oldest supported Cilium version.
+	warnWildcardToFromEndpointMessage = "It seems you have a CiliumClusterwideNetworkPolicy " +
+		"with a wildcard to/from endpoint selector. The behavior of this selector has been " +
+		"changed. The selector now only allows traffic to/from Cilium managed K8s endpoints, " +
```

Comment:
```suggestion
		"has been changed from 1.9 onwards. The selector now only allows traffic to/from Cilium managed K8s endpoints, " +
```

nit: I think it would be useful to mention from what version behavior changed. Based on the comment above, I'm assuming this changed on 1.9? (feel free to change the phrasing, mine was just a quick suggestion)

Reply:
We are still discussing the backport scenario for the Pull Request. I will update the comment if required once we decide on that. Thanks for pointing it out though.

Reply:
We have decided in the Cilium meeting that this PR is going to be backported to 1.7 and 1.8 to avoid having different implementations of the policy in different Cilium versions.